nawab.py

```diff
@@ -52,26 +57,63 @@ def wrapper(func, args, res):
 
 
 def main():
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument("-r", "--retweet", help="Retweet all tweets automatically, doesn't spawn a telegram bot",
+                        action='store_true', required=False)
+    parser.add_argument("-b", "--blacklist",type=list, required=False, help="Blacklist the given username")
+    parser.add_argument("-p", "--path", type=str, required=False,
+                        help="Path where the log files be stored. Note to create directory in that path beforehand.")
+    parser.add_argument('-V', '--verbose',action="store_const", const=20)
+    parser.add_argument('-s', '--silent', action="store_const", const=50)
```

Comment:
What was this const about again?? Is it needed now?

Reply:
@Aniketh01  its not const per say, it only act as reference for the conditions to work properly and also only when the user provide with the argument then only the value gets stored in args['verbose'] and it makes it easier to do the comparison.

Reply:
Ok. Got it. Thank for the clarification. Would you mind adding few lines as a comment for future reference?